integration_tests/dbt_project.yml

```diff
@@ -9,7 +9,7 @@ dispatch:
     
 vars: 
   apple_search_ads__using_search_terms: True
-  twitter_ads__using_keywords: False
+  twitter_ads__using_keywords: True
```

Comment:
Probably want to remove this edit from the PR once it is moved from draft to ready for review.

Reply:
removed

Reply:
i think this is just leftover from seeding the data into snowflake